make/tasks/Makefile.muxed

```diff
@@ -0,0 +1,18 @@
+# Copyright 2004-2015 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CHPL_MAKE_THREADS=soft-threads
```

Comment:
Yep, that's it.  The cleanest way to deal with the Makefile.muxed situation is to add it to the overlay in the Cray repo and then update the overlay and rm-overlay scripts to add and rm the symbolic link, and also I guess update the thing that cleans up after building the module.  Unfortunately (as you can tell from that list) that's also more work!  I would be willing to entertain putting a make/tasks/Makefile.muxed directly into the public repo instead, if people didn't feel it would be weird mentioning a tasking layer whose source isn't public.  (Especially since, if we can get Qthreads doing the necessary stuff, we're going to get rid of muxed entirely.)  Maybe do the latter as an expedient solution now, and if people seem put out, do it the hard way later?